tests/foreman/ui/test_user.py

```diff
@@ -258,6 +258,52 @@ def test_positive_create_with_all_roles(self):
                 self.assertIsNotNone(self.user.wait_until_element(
                     (strategy, value % role)))
 
+    @tier2
+    def test_positive_create_with_multiple_roles_and_login(self):
+        """Create User with multiple roles and re-login into application
+        using that user
+
+        @id: b1d96e56-42e0-4f4b-8f38-98fb72be5c64
+
+        @Assert: User is created successfully and can be used to login into
+        application without raising any errors. Also user can review his own
+        profile data.
+
+        @BZ: 1392513
```

Comment:
We don't have clear agreement, by imo it's better to reference it. Sometimes BZ tests follow really weird or invalid scenarios and without reference to specific BZ later on we may think the test is invalid and should be dropped. Also if the test will suddenly start to fail we'll have a reference to issue which should be reopened. And as we're referencing the issue in docstring it doesn't introduce any redundant code or performance degradation, so why not?
`@skip_if_bug_open` decorator should definitely be dropped after BZ is closed, but personally i can't see any serious drawback in leaving the link in docstring, just few small bonuses for us.